integration-tests/src/test/java/oracle/weblogic/kubernetes/ItClusterResourceScaling.java

```diff
@@ -0,0 +1,527 @@
+      logger.info("externalRestHttpsPort {0}", externalRestHttpsPort);
+      if (!isDomainRunning) {
+        logger.info("Installing and verifying domain");
+        assertTrue(createVerifyDomain(domainNamespace, domainUid),
+            "can't start or verify domain in namespace " + domainNamespace);
+        isDomainRunning = true;
+      }
+
+      // scale domain
+      int replicaCountdomain = 2;
+      String secretToken = getServerToken(opServiceAccount);
+      // decode the secret encoded token
+      String decodedToken = new String(Base64.getDecoder().decode(secretToken));
+      assertTrue(scaleClusterWithRestApi(domainUid, clusterName,replicaCountdomain + 1,
+              externalRestHttpsPort,opNamespace, decodedToken,"",
+              true, true),
+          "domain in namespace " + domainNamespace + " scaling operation failed");
+
+      String managedServerPodName2 = domainUid + managedServerPrefix + (replicaCountdomain + 1);
+      logger.info("Checking that the managed server pod {0} exists in namespace {1}",
+          managedServerPodName2, domainNamespace);
+      assertDoesNotThrow(() -> checkPodExists(managedServerPodName2, domainUid, domainNamespace),
+          "operator failed to manage domain, scaling was not succeeded");
+      ++replicaCountdomain;
+      logger.info("domain scaled to " + replicaCountdomain + " servers");
+
+      // decode the secret encoded token
+      String decodedTokenBad = new String(Base64.getDecoder().decode(secretToken)) + "badbad";
+      logger.info("Testing {0}", negativeTestCase1);
+      badAuthFailed = scaleClusterWithRestApi(domainUid, clusterName,replicaCountdomain + 2,
+              externalRestHttpsPort,opNamespace, decodedTokenBad, "401 Unauthorized", true,
+              true);
+      if (!badAuthFailed) {
+        failedNegativeTestCases.append(negativeTestCase1);
+      }
+      logger.info("Testing {0}", negativeTestCase2);
+      invalidDomainUidFailed = scaleClusterWithRestApi(domainUid + "invalid", clusterName,replicaCountdomain + 2,
+              externalRestHttpsPort,opNamespace, decodedToken,  "404 Not Found", true,
+              true);
+      if (!invalidDomainUidFailed) {
+        failedNegativeTestCases
+            .append(", ")
+            .append(negativeTestCase2);
+      }
+      logger.info("Testing {0}", negativeTestCase3);
+      invalidClusterNameFailed = scaleClusterWithRestApi(domainUid, clusterName + "invalid",replicaCountdomain + 2,
+              externalRestHttpsPort,opNamespace, decodedToken,  "404 Not Found", true,
+              true);
+      if (!invalidClusterNameFailed) {
+        failedNegativeTestCases
+            .append(", ")
+            .append(negativeTestCase3);
+      }
+      logger.info("Testing {0}", negativeTestCase4);
+      invalidReplicaNumberFailed = scaleClusterWithRestApi(domainUid, clusterName,replicaCountdomain + 12,
+          externalRestHttpsPort,opNamespace, decodedToken,  "400 Requested scaling count of 15"
```

Comment:
add space before opNamespace, same for others

Reply:
done